processes/ingest/met.py

```diff
@@ -99,38 +101,3 @@ def get_cover_path(self, file_type: Optional[str], record_id: str) -> Optional[s
                 return None
         
         return f'api/singleitem/image/pdf/p15324coll10/{record_id}/default.png'
-
-    def store_pdf_manifest(self, record: Record):
-        record_id = record.source_id.split('|')[0]
-        pdf_part = next(filter(lambda part: part.file_type == 'application/pdf', record.get_parts()), None)
-
-        if pdf_part is not None:
-            manifest_path = f'manifests/{record.source}/{record_id}'
-            manifest_url = get_stored_file_url(storage_name=self.s3_bucket, file_path=manifest_path)
-            manifest_json = self.generate_manifest(record=record, source_uri=pdf_part.url, manifest_uri=manifest_url)
-
-            self.s3_manager.createManifestInS3(manifestPath=manifest_path, manifestJSON=manifest_json, s3_bucket=self.s3_bucket)
-
-            record.has_part.insert(0, Part(
-                index=pdf_part.index,
-                url=manifest_url,
-                source=record.source,
-                file_type='application/webpub+json',
-                flags=pdf_part.flags
```

Comment:
Interestingly, the manifest flags for MET were that of the PDF flags which were download only as True. 

Should the PDF manifest only have the reader flag as True in all cases?

Reply:
The PDF manifest reader flag must always be true in all cases to allow the manifest to be accessed by the web reader.